imports/plugins/core/checkout/client/templates/checkout/completed/completed.js

```diff
@@ -20,6 +20,22 @@ Template.cartCompleted.helpers({
     }
     return false;
   },
+  ifUserOrder: function () {
```

Comment:
Let's not start a function name with `if`. If it returns a boolean you want it to start with `is`.  `+{{#if ifUserOrder}}` is some awkward syntax.